lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -262,46 +189,11 @@ class Chef::Knife::VsphereVmClone < Chef::Knife::BaseVsphereCommand
                  v
                }
 
-  option :run_list,
-         short: "-r RUN_LIST",
-         long: "--run-list RUN_LIST",
-         description: "Comma separated list of roles/recipes to apply",
-         proc: ->(o) { o.split(/[\s,]+/) },
-         default: []
-
   option :secret_file,
```

Comment:
This option is coming from chef,  `data_bag_secret_options.rb` file so can be removed from here.

Reply:
removed `secret_file` option.